src/data/lovelace.ts

```diff
@@ -20,6 +20,10 @@ export interface LovelaceViewConfig {
   background?: string;
 }
 
+export interface LovelaceBadgeConfig {
+  type?: string;
+}
```

Comment:
```suggestion
  [key: string]: any;
}
```

Reply:
Why add this?

Reply:
Because technically we don't know what there might be in it, being custom elements etc.